core/embed/rust/src/ui/model_tt/component/title.rs

```diff
@@ -0,0 +1,80 @@
+use super::theme;
+use crate::ui::{
+    component::{Child, Component, ComponentExt, Event, EventCtx},
+    display,
+    geometry::{Insets, Rect},
+};
+
+pub struct Title<T, U> {
```

Comment:
Will rename it to `Frame`. Not sure if I understand correctly but if you don't need a title you can just use the thing directly without a wrapper.

Reply:
i mean more like, "Frame" is a default top-level component for every "layout" (where "layout" is a screen as designed in Figma). But in case we have layouts that don't have title, we might need a different top-level component in case just sticking, e.g., a ButtonPage, is not appropriate. (consider a pop-up that is a grey bubble on black background)

Reply:
Fixed in 32e5653c2e53c24f0979d36cc39d3d30f5f6fa82 and be6ecd2b9848979c589a79c28b43d0ced9ff376a.